### DISCUSSION TITLE
Roborock s5 MAX, Getting map coordinates.

### DISCUSSION THREAD

**Discussion by Tontze** [rytilahti/python-miio#984] [Q&A] [answered]
Is there anyway to pull map for Roborock s5 MAX from cloud or something ? Would like to use zoned cleaning...

**Comment by Tontze** [comment, marked as answer]
Well, openhab had tool for this :)

**Reply by marcelrv** [reply]
He is refering to this: 
https://community.openhab.org/t/xiaomi-vacuum-map-viewer-to-find-coordinates-for-zone-cleaning/103500/1

**Reply by Tontze** [reply]
Yes sry, i thought i added link ..  Awesome app !

**Reply by Hukuma1** [reply]
> He is refering to this: https://community.openhab.org/t/xiaomi-vacuum-map-viewer-to-find-coordinates-for-zone-cleaning/103500/1

Any chance there's a way to use your awesome tool with Home Assistant? Or from Mi Home/Roborock app? :(

Edit: Answering my own question: https://github.com/da-mkay/mihome-vacuum-map-dl with your tool works like a charm. Thank you!
